pkg/local_object_storage/metabase/status.go

```diff
@@ -64,3 +79,59 @@ func (db *DB) ObjectStatus(address oid.Address) (ObjectStatus, error) {
 	res.Error = err
 	return res, err
 }
+
+func readBuckets(tx *bbolt.Tx, cID cid.ID, objKey []byte) []BucketValue {
+	var res []BucketValue
+	cIDRaw := containerKey(cID, make([]byte, cidSize))
+
+	objectBuckets := [][]byte{
+		graveyardBucketName,
+		garbageObjectsBucketName,
+		toMoveItBucketName,
+		primaryBucketName(cID, make([]byte, bucketKeySize)),
+		bucketNameLockers(cID, make([]byte, bucketKeySize)),
+		storageGroupBucketName(cID, make([]byte, bucketKeySize)),
+		tombstoneBucketName(cID, make([]byte, bucketKeySize)),
+		smallBucketName(cID, make([]byte, bucketKeySize)),
+		rootBucketName(cID, make([]byte, bucketKeySize)),
+		parentBucketName(cID, make([]byte, bucketKeySize)),
+		linkObjectsBucketName(cID, make([]byte, bucketKeySize)),
+		firstObjectIDBucketName(cID, make([]byte, bucketKeySize)),
+	}
+
+	for _, bucketKey := range objectBuckets {
+		b := tx.Bucket(bucketKey)
+		if b == nil {
+			continue
+		}
+
+		res = append(res, BucketValue{
+			BucketIndex: int(bucketKey[0]), // the first byte is always a prefix
```

Comment:
1st i read ur comment, then looked into `VERSION.md` file: seems like it contains incorrect info. For example:
```
- Bucket mapping first object ID to a list of objects IDs
  - Name: container ID + `19`
```
to me it's unclear that `19` is a prefix (although it's it), more like a suffix. I suggest to fix docs